README.md

```diff
@@ -121,7 +121,7 @@ To build a specific release, use the command `release --platform="win64"` for ex
 
 ### Running with debug | Inspector
 
-To be able to open Inspector, you will need SDK flavours of NW.js
```

Comment:
This is not a spelling mistake. Flavour is spelt correctly.

Reply:
Okkk, i will change it